.gitignore

```diff
@@ -33,6 +33,7 @@ R/pkg/tests/fulltests/Rplots.pdf
 build/*.jar
 build/apache-maven*
 build/scala*
+build/zinc-*
```

Comment:
We don't need this change. see https://github.com/apache/spark/pull/31777#discussion_r589369659

Reply:
Thanks! Let me close the PR.